### PR TITLE
Overloaded Functions

### PR DESCRIPTION
**Pull request by el5846**
Added in functions for unixTimestamp of no parameters and date parameter, round no parameter, and subdue with days parameter. Fixed minor style errors.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Ports
-WARP_CORE_MYSQL_PORT=3307
+WARP_CORE_MYSQL_PORT=3308
```
Comment (tsoppet) [inline .env]: did you want to make this change to the .env?

Comment (el5846) [reply .env]: Tomas helped me set it up so I can work on Vulcan and warp-core. Can we change it back at the end?

Comment (rooeque) [reply .env]: let's change this back before the merge

File: .env [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Ports
 WARP_CORE_MYSQL_PORT=3307
 WARP_CORE_GRAFANA_PORT=3000
-WARP_CORE_INFLUX_PORT=8086
+WARP_CORE_INFLUX_PORT=8087
```
Comment (rooeque) [inline .env]: please change this port back as well

**Comment by tomnis** [comment]
@el5846 anything else you want to add here?

**Comment by el5846** [comment]
Sorry I forgot to overload one thing

File: src/main/scala/com/workday/warp/persistence/mysql/HasWarpSlickDsl.scala [hunk 3]
```diff
@@ -29,25 +30,54 @@ trait HasWarpSlickDsl {
...
+        queryBuilder.sqlBuilder += ")"
+      }
+      expression.apply(str)
+    }
+
+    /**
+      * Translates to YEAR(string) function
+      * @return just the year in Int
+      */
+    def yearString(): Rep[Int] = {
```
Comment (rooeque) [inline src/main/scala/com/workday/warp/persistence/mysql/HasWarpSlickDsl.scala]: Can we rename these back to `year` to follow MySQL function names?

File: dependencies_2.12.lock [hunk 4]
```diff
@@ -5800,5 +5800,52 @@
             "locked": "1.3.6",
             "requested": "1.3.6"
         }
+    },
+    "zinc": {
```
Comment (rooeque) [inline dependencies_2.12.lock]: hmm where is this coming from? We're pulling in scala 2.10 as well

@tomnis

Comment (tomnis) [reply dependencies_2.12.lock]: hmm its in the zinc configuration... this has to do with gradle scala plugin resolving scalac. i think this is ok. scalac itself needs a scala-library that may be different from what our library is using
https://docs.gradle.org/current/userguide/scala_plugin.html#sec:configure_zinc_compiler

however, dependency lock files should be updated in lock-step (the 2.11 file as well).

@el5846 can you back out this change to 2.12 lockfile?

**Pull request merged**